### PR TITLE
fix: task manager should continue not exit if some task gen sub taks …

### PR DESCRIPTION
**Pull request by liuyu85cn**
fix: task manager should continue not exit if some task gen sub taks …

### CODE REVIEW

File: src/storage/admin/AdminTaskManager.cpp [hunk 1]
```diff
@@ -111,10 +111,11 @@ void AdminTaskManager::schedule() {
         auto task = it->second;
         auto errOrSubTasks = task->genSubTasks();
         if (!nebula::ok(errOrSubTasks)) {
-            LOG(ERROR) << "genSubTasks() failed=" << static_cast<int>(nebula::error(errOrSubTasks));
+            auto code = static_cast<int>(nebula::error(errOrSubTasks));
+            LOG(ERROR) << "genSubTasks() failed, err=" << code;
```
Comment (panda-sheep) [inline src/storage/admin/AdminTaskManager.cpp:115]: A little suggestion,
Should I print the task name of the task into the log?

Comment (liuyu85cn) [reply src/storage/admin/AdminTaskManager.cpp]: good catch, fixed

**Review by jievince — APPROVED** [review]
LGTM!

**Review by panda-sheep — APPROVED** [review]
Excellent!

**Pull request merged**